### ISSUE TITLE
AWS amplify deploy completes successfully but error 404 on access to website

### ISSUE THREAD

**Issue by AnonymousAmalgrams** [aws-amplify/amplify-hosting#3736]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-hosting/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-hosting/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

### App Id

dqcry7h9xp57z

### AWS Region

us-east-1

### Amplify Hosting feature

Deployments, SSR

### Frontend framework

Next.js

### Next.js version

13.5.3

### Next.js router

App Router

### Describe the bug

Deploys successfully but no content is actually pushed to the website. Domain name and DNS are confirmed to be valid and the page responds to dns queries sent by route53. Updated platform to "platform": "WEB_DYNAMIC" with no result.

### Expected behavior

Expect content to be served correctly during deployment and appear when navigating to the site.

### Reproduction steps

1. Deploy with AWS Amplify.
2. Navigate to domain name https://coal-reality.me/
3. ??????

### Build Settings

```yaml
version: 1
frontend:
  phases:
    preBuild: 
      commands: 
        - cd coal-reality
        - npm ci
    # IMPORTANT - Please verify your build commands
    build:
      commands: 
        - npm run build
  artifacts:
    # IMPORTANT - Please verify your build output directory
    baseDirectory: coal-reality/.next/
    files:
      - '**/*/*/*'
  cache:
    paths:
        - 'coal-reality/node_modules/**/*' # Cache `node_modules` for faster `yarn` or `npm i`
        - 'coal-reality/.next/cache/**/*' # Cache Next.js for faster application rebuilds

backend:
```

### Log output

<details>

```
# Put your logs below this line
No backend:

Frontend:
                                 # Starting phase: preBuild
                                 # Executing command: cd coal-reality
2023-09-28T11:27:17.365Z [INFO]: # Executing command: npm ci
2023-09-28T11:27:41.179Z [INFO]: added 498 packages, and audited 499 packages in 24s
2023-09-28T11:27:41.182Z [INFO]: 110 packages are looking for funding
                                 run `npm fund` for details
                                 found 0 vulnerabilities
2023-09-28T11:27:41.201Z [INFO]: # Completed phase: preBuild
                                 # Starting phase: build
2023-09-28T11:27:41.201Z [INFO]: # Executing command: npm run build
2023-09-28T11:27:41.549Z [INFO]: > coal-reality@0.1.0 build
                                 > next build
2023-09-28T11:27:42.198Z [INFO]: Creating an optimized production build ...
2023-09-28T11:27:45.111Z [WARNING]: Warning: For production Image Optimization with Next.js, the optional 'sharp' package is strongly recommended. Run 'npm i sharp', and Next.js will use it automatically for Image Optimization.
                                    Read more: https://nextjs.org/docs/messages/sharp-missing-in-production
2023-09-28T11:27:57.438Z [WARNING]: (node:965) [DEP_WEBPACK_MODULE_UPDATE_HASH] DeprecationWarning: Module.updateHash: Use new ChunkGraph API
                                    (Use `node --trace-deprecation ...` to show where the warning was created)
2023-09-28T11:28:00.089Z [INFO]: ✓ Compiled successfully
2023-09-28T11:28:00.090Z [INFO]: Linting and checking validity of types ...
2023-09-28T11:28:02.468Z [INFO]: ./app/coal-mines/CoalMineInstance.tsx
                                 24:17  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
                                 67:25  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
                                 67:25  Warning: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  jsx-a11y/alt-text
                                 ./app/counties/CountiesInstance.tsx
                                 24:17  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
                                 67:25  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
                                 67:25  Warning: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  jsx-a11y/alt-text
                                 ./app/healthcare-providers/HealthcareProvidersInstance.tsx
                                 24:17  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
                                 67:25  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
                                 67:25  Warning: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.  jsx-a11y/alt-text
                                 info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
2023-09-28T11:28:04.022Z [INFO]: Collecting page data ...
2023-09-28T11:28:04.960Z [INFO]: Generating static pages (0/18) ...
2023-09-28T11:28:05.146Z [INFO]: Generating static pages (4/18)
2023-09-28T11:28:05.218Z [INFO]: Generating static pages (8/18)
2023-09-28T11:28:05.305Z [INFO]: Generating static pages (13/18)
2023-09-28T11:28:05.367Z [INFO]: ✓ Generating static pages (18/18)
2023-09-28T11:28:05.379Z [INFO]: Finalizing page optimization ...
2023-09-28T11:28:05.383Z [INFO]: 
2023-09-28T11:28:05.393Z [INFO]: Route (app)                                      Size     First Load JS
                                 ┌ ○ /                                            181 B          79.6 kB
                                 ├ ○ /_not-found                                  878 B          80.3 kB
                                 ├ ○ /about                                       23.9 kB         103 kB
                                 ├ ○ /coal-mines                                  3.59 kB        89.8 kB
                                 ├ ○ /coal-mines/coal-mine-1                      1.11 kB         111 kB
                                 ├ ○ /coal-mines/coal-mine-2                      1.13 kB         111 kB
                                 ├ ○ /coal-mines/coal-mine-3                      1.12 kB         111 kB
                                 ├ ○ /counties                                    3.69 kB        89.9 kB
                                 ├ ○ /counties/counties-1                         1.24 kB         111 kB
                                 ├ ○ /counties/counties-2                         1.28 kB         111 kB
                                 ├ ○ /counties/counties-3                         1.4 kB          111 kB
                                 ├ ○ /healthcare-providers                        3.93 kB        90.1 kB
                                 ├ ○ /healthcare-providers/healthcare-provider-1  1.13 kB         111 kB
                                 ├ ○ /healthcare-providers/healthcare-provider-2  1.3 kB          111 kB
                                 └ ○ /healthcare-providers/healthcare-provider-3  1.31 kB         111 kB
                                 + First Load JS shared by all                    79.5 kB
                                 ├ chunks/864-975492c2867153e8.js               26.6 kB
                                 ├ chunks/fd9d1056-5b3ee06fda64bfbf.js          50.8 kB
                                 ├ chunks/main-app-f96c3abf373a3af4.js          221 B
                                 └ chunks/webpack-d2c94483f6ce9836.js           1.87 kB
2023-09-28T11:28:05.394Z [INFO]: ○  (Static)  automatically rendered as static HTML (uses no initial props)
2023-09-28T11:28:05.529Z [INFO]: # Completed phase: build
2023-09-28T11:28:05.531Z [INFO]: ## Build completed successfully
2023-09-28T11:28:05.532Z [INFO]: # Starting caching...
2023-09-28T11:28:05.544Z [INFO]: # Creating cache artifact...
2023-09-28T11:28:18.716Z [INFO]: # Created cache artifact
2023-09-28T11:28:18.902Z [INFO]: # Uploading cache artifact...
2023-09-28T11:28:22.556Z [INFO]: # Uploaded cache artifact
2023-09-28T11:28:22.772Z [INFO]: # Caching completed
2023-09-28T11:28:22.779Z [INFO]: # No custom headers found.
2023-09-28T11:28:22.805Z [INFO]: # Starting build artifact upload process...
2023-09-28T11:28:25.795Z [INFO]: # Uploading build artifact '__artifacts.zip'...
2023-09-28T11:28:25.839Z [INFO]: # Uploading build artifact '__artifactsHash.zip'...
2023-09-28T11:28:26.287Z [INFO]: # Build artifact upload completed
2023-09-28T11:28:26.287Z [INFO]: # Starting environment caching...
2023-09-28T11:28:26.288Z [INFO]: # Uploading environment cache artifact...
2023-09-28T11:28:26.363Z [INFO]: # Uploaded environment cache artifact
2023-09-28T11:28:26.363Z [INFO]: # Environment caching completed
Terminating logging...

Deploy:
2023-09-28T11:28:33 [INFO]: Beginning deployment for application dqcry7h9xp57z, branch:main, buildId 0000000088
2023-09-28T11:28:33 [INFO]: Got archive: 31496477 bytes
2023-09-28T11:28:37 [INFO]: Deployment complete


```

</details>

### Additional information

Gitlab repo:
https://gitlab.com/emily.yue/group-3-swe-project

**Comment by andresdameson** [comment]
I had the same problem and I have fixed it downgrading NextJJ to 13.4.12

**Comment by AnonymousAmalgrams** [comment]
my issue was relating to the directory structure of my build file
